### PR TITLE
Inject logger into exo-clone

### LINKED ISSUE #124: Inject logger into subclasses of exo commands
Instead of emitting a bunch of events, sometimes through multiple layers, the logger should be injected to classes of the exo commands so that they can call it directly. 

Check off once completed:
- [x] exo-clone
- [x] exo-lint
- [x] exo-sync
- [x] exo-test

### PR DESCRIPTION
**Pull request by trushton**
resolves #124  

@kevgo

### CODE REVIEW

File: exo-clone/src/cli.ls [hunk 1]
```diff
@@ -13,20 +13,11 @@ clone = ->
 
   [_, _, repo-origin] = process.argv
   return missing-origin! unless repo-origin
-  repo = repo-info repo-origin
+  repository = repo-info repo-origin
```
Comment (kevgo) [inline exo-clone/src/cli.ls:16]: You renamed this variable only in one place in this file. Do you even use it?

File: exo-clone/src/service-cloner.ls [hunk 2]
```diff
@@ -35,7 +36,7 @@ class ServiceCloner extends EventEmitter
 
 
   write: (text) ~>
-    @emit 'output', {@name, text: text.trim!.replace /\.*$/, ''}
+    @logger.log {@name, text: (text.trim!.replace /\.*$/, '')}
```
Comment (kevgo) [inline exo-clone/src/service-cloner.ls]: Please use parentheses for normal function calls: 

```ls
text: text.trim!.replace(/\.*$/, '')
```

and not 

```
text: (text.trim!.replace /\.*$/, '')
```

We aren't writing LISP or Scheme here  :)

**Review by kevgo — APPROVED** [review]

File: exo-clone/src/cli.ls (hunk 1 above)
Comment (trushton) [reply exo-clone/src/cli.ls:16]: The `repository` referred to here is the object that is passed into the `AppCloner`

Comment (kevgo) [reply exo-clone/src/cli.ls:16]: Ah, right! I really need to take it easy today, not fit..

**Review by kevgo — APPROVED** [review]

**Pull request closed without merging**